blocks/bloom/filter.go

```diff
@@ -15,6 +15,7 @@ type Filter interface {
 	Find([]byte) bool
 	Merge(Filter) (Filter, error)
 	HammingDistance(Filter) (int, error)
+	SupersetOf(Filter) (bool, error)
```

Comment:
I'm thinking this is the wrong way to add these extensions-- they're putting everything into a single interface. I think maybe the `container/heap` -- http://golang.org/pkg/container/heap/ -- approach is the right one.

So: 
```go
func Superset(a, b Filter) (bool, error) {
  // return whether a is superset of b
}
```